scripts/jira_tickets/jira_recurrent_tickets.vertebrates.json

```
@@ -221,6 +221,42 @@
             "summary": "Import the Alt-allele groups",
             "name_on_graph": "Alt-alleles import"
          },
+         {
+             "component": "Production tasks",
+             "description": "https://github.com/Ensembl/ensembl-compara/blob/release/97/docs/production/READMEs/pair_aligner_patches.rst",
+             "summary": "Merge LastZ_Patch human alignments",
```

The wording here doesn't quite capture the task at hand.
```suggestion
             "summary": "Run LastZ_PATCH alignments for human",
```